stagingUtils.js

```diff
@@ -197,13 +199,8 @@ module.exports = {
 
   async getUpstreamRepo() {
     try {
-      const forkConfig = (await exec('git remote -v')).stdout;
-      const configArray = forkConfig.split('\n');
-
-      let upstreamRepo = (configArray[2].replace('upstream', ''));
-      upstreamRepo = upstreamRepo.split(/(\S)+[:]((\S)+)/);
-      upstreamRepo = upstreamRepo[2];
-
+      const forkConfig = (await exec('git remote get-url upstream')).stdout;
```

Comment:
what if the inner code fails for some reason? then .stdout might fail?

Reply:
being inside a try/catch, doesn't that safe guard?